include/ocpp/v16/charge_point.hpp

```diff
@@ -295,12 +295,14 @@ class ChargePoint : ocpp::ChargePoint {
     void on_max_current_offered(int32_t connector, int32_t max_current);
 
     /// \brief Notifies chargepoint that a new session with the given \p session_id has been started at the given \p
-    /// connector with the given \p reason
-    void on_session_started(int32_t connector, const std::string& session_id, const std::string& reason);
+    /// connector with the given \p reason . The logs of the session will be written into \p session_logging_path if
+    /// present
+    void on_session_started(int32_t connector, const std::string& session_id, const std::string& reason,
```

Comment:
I think these API changes necessitate a version increase and the 1.6 OCPP module in everest-core needs to be updated as well

Reply:
I added it. Propably needs to change in this PR: https://github.com/EVerest/libocpp/pull/71 depending on what we merge first